packages/off-chain/shell/copy_merkle_tree.sh

```diff
@@ -0,0 +1,19 @@
+# Shell script to copy merkle tree file from hardhat package
```

Comment:
can we not just import the function from the other package?

Reply:
we can, but it won't work on docker image because, hardhat(the one you are working on) package is not on npm registry hence, when docker is initialising, it will not find getMerkleTree file and fail.